src/components/HeaderAll.vue

```diff
@@ -1,11 +1,16 @@
 <script setup lang="ts">
 import StopWatch from "./StopWatch.vue";
+import { timerShowStore } from "../stores/timerShow";
+import { storeToRefs } from "pinia";
+
+const timerShow = timerShowStore();
+const { getShowState } = storeToRefs(timerShow);
 </script>
 
 <template>
   <div class="flex px-4 py-4 bg-gray-400 items-center justify-between">
     <div class="px-2 font-bold text-2xl">CODE-TYPING</div>
-    <StopWatch />
+    <StopWatch v-show="getShowState" />
```

Comment:
script内での取得できなかったです...
useRoute.nameで"/"になります( ；∀；)

```suggestion
    <StopWatch v-show="$route.name == 'game'" />
```

Reply:
[443ac20](https://github.com/recursion-team-o/typing-game/pull/58/commits/443ac209370e2490766e6b56de8d3fdc92f5015f)